src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  width: $card-width;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  color: $header-text-color;
+  font-size: $font-basic-size;
+  line-height: 14px;
+
+  &__image {
+    display: flex;
+    margin: auto;
+    margin-bottom: 40px;
+  }
+
+  &__tittle {
+    text-align: left;
+    margin-bottom: 4px;
+    line-height: 18px;
+    font-size: $font-basic-size;
+    font-weight: $roboto-medium;
+  }
+
+  &__review,
+  &__code,
+  &__price {
+    font-weight: 400;
+  }
+
+  &__review,
+  &__code {
+    font-size: 10px;
+  }
+
+  &__code,
+  &__price {
+    color: $card-price-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: $roboto-bold;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 12px 0;
+    font-size: 14px;
+    text-align: center;
+    background: $v1-button-color;
```

Comment:
hmm, I wonder why `v1`, it's better not to use the component name for variables with color (we can use the same color for a completely different component), just set the name the same way we name the color

F.e.

```SCSS
$c-red
$c-light-blue
$c-dark-gray
```

or we can use color saturation in the name
<img width="144" alt="image" src="https://user-images.githubusercontent.com/86183007/216517467-c9b38dd8-5ad4-4ea3-8960-e58b36391ec7.png">